test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSRunner.scala

```diff
@@ -214,6 +223,12 @@ final class ScalaJSRunner private[testadapter] (
       throw new IllegalStateException("Runner is already done")
   }
 
+  private def createAnticipatedSlaves(): Unit = {
+    val count = Runtime.getRuntime().availableProcessors()
+    for (_ <- 0 to count)
```

Comment:
Shouldn't this be `0 until count` or `1 to count`? If `0 to count` is correct, it probably needs a comment telling why we need 1 slave more than the number of available processors.